Misc update README

sa6ta6ni6c opened the pull request.

Commit:
Misc update

pokamest commented on README.md:
```diff
@@ -59,21 +59,21 @@ Look deploy folder for build scripts.
 export PATH=$PATH:~/go/bin
 go install golang.org/x/mobile/cmd/gomobile@latest
-gomobile init
+go mobile init
```
Correct is `gomobile init`
https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile

pokamest commented on README.md:
````diff
@@ -59,21 +59,21 @@ Look deploy folder for build scripts.
 ```bash
 export QT_BIN_DIR="<PATH-TO-QT-FOLDER>/Qt/<QT-VERSION>/ios/bin"
-export QT_MACOS_ROOT_DIR="<PATH-TO-QT-FOLDER>/Qt/<QT-VERSION>/macos"
+export QT_MACOS_ROOT_DIR="<PATH-TO-QT-FOLDER>/Qt/<QT-VERSION>/macOS"
````
Default Qt path is `"<PATH-TO-QT-FOLDER>/Qt/<QT-VERSION>/macos"`
Please revert

pokamest commented on README.md:
```diff
@@ -45,11 +45,11 @@ Want to contribute? Welcome!
 1. First, make sure you have [XCode](https://developer.apple.com/xcode/) installed, at least version 14 or higher.
 
-2. We use QT to generate the XCode project. we need QT version 6.6.1. Install QT for macos in [here](https://doc.qt.io/qt-6/macos.html) or [QT Online Installer](https://www.qt.io/download-open-source). Required modules:
+2. We use QT to generate the XCode project. We need QT version 6.6.1. Install QT for Macos [here](https://doc.qt.io/qt-6/macos.html) or [QT Online Installer](https://www.qt.io/download-open-source). Required modules:
```
It's better to write `MacOS` everywhere since you started to fix it ))

pokamest commented:
Thank you for the contributing!

Commit:
fix

Need more clarification on this:

6. Open the XCode project. You can then run /test/archive/ship the app.

Commit:
Update README.md
